src/preferences/dialog/dlgprefmixer.h

```diff
@@ -73,9 +76,23 @@ class DlgPrefEQ : public DlgPreferencePage, public Ui::DlgPrefEQDlg  {
 
     bool eventFilter(QObject* obj, QEvent* e) override;
 
+    UserSettingsPointer m_pConfig;
+
+    QGraphicsScene* m_pxfScene;
+
+    // X-fader values
+    double m_xFaderMode, m_transform, m_cal;
+
+    PollingControlProxy m_mode;
+    PollingControlProxy m_curve;
+    PollingControlProxy m_calibration;
+    PollingControlProxy m_reverse;
+    PollingControlProxy m_crossfader;
```

Comment:
does somebody know why these are `PollingControlProxy`s?

Reply:
because they don't require signal connections, just read and set on demand in update/apply slots

Reply:
Right, I forgot how PollingControlProxy actually works (I thought it emulates signals and slots by continuously polling a CO).